<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenuGUI</class>
 <widget class="QMainWindow" name="MainMenuGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainMenuGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="yearSelect">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>77</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>71</y>
      <width>32</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Year:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>118</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Semester:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>166</y>
      <width>37</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Class:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>213</y>
      <width>50</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Section:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>261</y>
      <width>52</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Student:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Lab Number:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Component</string>
    </property>
   </widget>
   <widget class="QListWidget" name="disjava">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>301</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="componentSelect">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>101</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="discomment">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>400</y>
      <width>411</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="semesterSelect">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>124</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Toggle Anonymous Grading</string>
    </property>
   </widget>
   <widget class="QComboBox" name="classSelect">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>171</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="sectionSelect">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>218</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="studentSelect">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>265</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Refresh">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QComboBox" name="labSelect">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>310</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="saveAndExit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>460</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save and Exit</string>
    </property>
   </widget>
   <widget class="QListWidget" name="disrubric">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>60</y>
      <width>301</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1005</width>
     <height>22</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAdd_New_Labs"/>
    <addaction name="actionComment"/>
    <addaction name="actionExport_Lab"/>
    <addaction name="actionAdd_Rubric"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAdd_New_Labs">
   <property name="text">
    <string>Add New Labs</string>
   </property>
  </action>
  <action name="actionComment">
   <property name="text">
    <string>Comment</string>
   </property>
  </action>
  <action name="actionExport_Lab">
   <property name="text">
    <string>Export Lab</string>
   </property>
  </action>
  <action name="actionAdd_Rubric">
   <property name="text">
    <string>Add Rubric</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
